cpmonitor/admin.py

```diff
@@ -55,6 +55,24 @@ def edit_tasks(self, city: City):
 
 
 class TaskAdmin(TreeAdmin):
+    fields = (
```

Comment:
I refrained from explicitly listing fields, here, in order to catch all fields not excluded. Is there a reason to list them, here? Which field is missing?

Reply:
This also defines the order of the fields displayed. If you only define readonly_fields, the readonly fields will be moved to the bottom.